Godot can't locate MSBuild

<!-- Please search existing issues for potential duplicates before filing yours:
https://github.com/godotengine/godot/issues?q=is%3Aissue
-->

**Godot version:**
<!-- Specify commit hash if non-official. -->
919fa75803177d0dc8f03c16b6be1360140145e1 Mono

**OS/device including version:**
<!-- Specify GPU model and drivers if graphics-related. -->
Ubuntu 18.10

**Issue description:**
<!-- What happened, and what was expected. -->
Godot crashes upon opening a project with the error:

```
WARNING: godot_icall_BuildInstance_get_MSBuildPath: Cannot find binary for 'MSBuild (Mono)'
   At: modules/mono/editor/godotsharp_builds.cpp:157.

Unhandled Exception:
System.IO.FileNotFoundException: Cannot find the MSBuild executable.
  at GodotSharpTools.Build.BuildInstance.GetMSBuildPath () [0x00013] in <3784b87b8dcf4df689003e241edd60cb>:0 
  at GodotSharpTools.Build.BuildInstance.Build (System.String loggerAssemblyPath, System.String loggerOutputDir, System.String[] customProperties) [0x0003d] in <3784b87b8dcf4df689003e241edd60cb>:0 
```

However, msbuild is installed, as I am able to build the editor with it, and can call it from the terminal:
```
Microsoft (R) Build Engine version 16.0.0.0 for Mono
Copyright (C) Microsoft Corporation. All rights reserved.

MSBUILD : error MSB1003: Specify a project or solution file. The current working directory does not contain a project or solution file.
```

--- a/drivers/unix/os_unix.cpp
+++ b/drivers/unix/os_unix.cpp
@@ -471,7 +471,7 @@ String OS_Unix::get_environment(const String &p_var) const {
 
 bool OS_Unix::set_environment(const String &p_var, const String &p_value) const {
 
-	return setenv(p_var.utf8().get_data(), p_var.utf8().get_data(), /* overwrite: */ true) == 0;
+	return setenv(p_var.utf8().get_data(), p_value.utf8().get_data(), /* overwrite: */ true) == 0;
 }
 
 int OS_Unix::get_processor_count() const {

